db/version_edit.cc

```diff
@@ -894,6 +918,11 @@ std::string VersionEdit::DebugJSON(int edit_num, bool hex_key) const {
       jw << "FileSize" << f.fd.GetFileSize();
       jw << "SmallestIKey" << f.smallest.DebugString(hex_key);
       jw << "LargestIKey" << f.largest.DebugString(hex_key);
+      if (f.min_timestamp != kDisableUserTimestamp) {
+        assert(f.max_timestamp != kDisableUserTimestamp);
+        jw << "MinTimestamp" << f.min_timestamp;
+        jw << "MaxTimestamp" << f.max_timestamp;
```

Comment:
Print timestamp in hex format?

Reply:
OK. now fix it to Slice(f.min_timestamp).ToString(true).